app/src/main/java/corp/kairos/adamastor/Onboarding/Onboard2SpecialPermissionActivity.java

```diff
@@ -38,6 +38,5 @@ public void showUsageSettings(View v){
 
     public void goNext() {
```

Comment:
Remove this method and inline the only line in it in the code. Calling a method for just one instruction isn't a good idea :+1: